pkgs/by-name/va/vapoursynth-zip/package.nix

```
@@ -0,0 +1,42 @@
+{
+  callPackage,
+  fetchFromGitHub,
+  lib,
+  stdenv,
+  zig,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vapoursynth-zip";
+  version = "7";
+
+  src = fetchFromGitHub {
+    owner = "dnjulek";
+    repo = "vapoursynth-zip";
+    tag = "R${finalAttrs.version}";
+    hash = "sha256-/bHD+t5upQ+So1k+5/IbNTfeugMqcnlDlaH/H9nXgR8=";
+  };
+
+  nativeBuildInputs = [
+    zig.hook
+  ];
```

```suggestion
  buildInputs = [
    zig.hook
  ];
```
Required for `strictDeps = true` support (#178468)